### ISSUE TITLE
quality of the GTs

### ISSUE THREAD

**Issue by lingcong-k** [ZhaoJ9014/Multi-Human-Parsing_MHP#41]
HI, thanks for this great dataset

I found some GTs are quite noisy, is there any bugs during encoding the GTs coz some noises seems quite buggy.
![image](https://user-images.githubusercontent.com/73646719/120195512-2d9d7280-c21f-11eb-8225-745411b53fa3.png)
![image](https://user-images.githubusercontent.com/73646719/120195536-35f5ad80-c21f-11eb-8570-15e71ea439c1.png)
![image](https://user-images.githubusercontent.com/73646719/120195578-41e16f80-c21f-11eb-9d3b-46808815e54e.png)

**Comment by b03505036** [comment]
@lingcong-k  
Hi, do you successful deal with the dataset?  Or, turn it into other format like COCO or Pascal?

**Comment by b03505036** [comment]
I got the same result of these pictures.

**Comment by lingcong-k** [comment]
@b03505036  yeah i guess no way to deal with it coz thats the quality of the labels.. i improved it a bit with some preprocessing but still noisy labels
